### DISCUSSION TITLE
Chinese chars > PDF

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#17540] [How to]
Gents, trying to get PDF template working with Chinese lang. Had set Noto Sans as per guidance, but printing garbage.
Noto should support many scripts, and so I wonder, has anything in this matter changed in 6.5? (recently upgraded)
Help appreciated

**Comment by garryatbrighton** [comment]
By the way, if I purchase PDF premium, will it support various scripts?

**Comment by ArniSamano** [comment]
Thing been raised few times here. My opinion is that if you are building online app (doesn't matter commercial or OS) and you not providing basics (such as multilingual support) you are sailing away form marketing common sense. Developer or common user who wants to deploy such app, and facing fundamental issue at the beginning of the journey,  will possibly feel a bit discouraged. 
Noticed also that developer quiet down for some reason, as open issues, discussions in most of the cases, hit a solid.
Would it be mysterious Yetiforce 7 a reason? Just saying...

**Comment by mariuszkrzaczkowski** [comment]
https://github.com/YetiForceCompany/YetiForceCRM/discussions/17549#discussioncomment-10169253
